File: Algorithms/217. Contains Duplicate/Solution.py
#coding=utf-8
__author__ = 'xuxuan'
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        map={}
        for num in nums:
            if map.get(num) is None:
                map[num]=1
            else :
                return True
        return False

File: Algorithms/217. Contains Duplicate/Solution.java
public class Solution {
    public boolean containsDuplicate(int[] nums) {
        Map<Integer,Integer> map=new HashMap<Integer,Integer>();
        for(int i=0;i<nums.length;i++){
            if(map.get(nums[i])==null){
                map.put(nums[i],1);
            }else{
                return true;
            }
        }
        return false;
    }
}